documentation/src/main/asciidoc/ch06.asciidoc

```diff
@@ -173,6 +173,19 @@ It is important to add each configured constraint violation by calling `addConst
 Only after that the new constraint violation will be created.
 ====
 
+[WARNING]
+====
+Note that the custom message template is passed directly to the Expression Language engine.
+
+Thus, you should be very careful when integrating user input in a custom message template as it will be interpreted
+by the Expression Language engine, which is usually not the behavior you expect and could cause a sensitive data leak.
+
+If you have this requirement, you should:
```

Comment:
Maybe "If you need to integrate user input"? I wasn't clear whether the requirement you were mentioning was "using custom message templates" or "allowing user input in custom message templates".

Reply:
Right, changed.